docs/core/run-time-config/networking.md

```diff
@@ -0,0 +1,28 @@
+---
+title: Networking config settings
+description: Learn about run-time settings that configure networking for .NET Core apps.
+ms.date: 11/27/2019
+ms.topic: reference
+---
+# Run-time configuration options for networking
+
+## HTTP/2 protocol
+
+- Configures whether support for the HTTP/2 protocol is enabled.
+- Default: Disabled (`false`).
+
+| | Setting name | Values | Introduced version |
+| - | - | - |
+| **runtimeconfig.json** | `System.Net.Http.SocketsHttpHandler.Http2Support` | `false` - disabled<br/>`true` - enabled | .NET Core 3.0 |
+| **Environment variable** | `DOTNET_SYSTEM_NET_HTTP_SOCKETSHTTPHANDLER_HTTP2SUPPORT` | 0 - disabled<br/>1 - enabled | .NET Core 3.0 |
+
+## Sockets HTTP handler
```

Comment:
We are removing these settings for .NET 5. Can we update the docs for this?

cc @ManickaP

Reply:
We aren't supposed to talk about future functionality in docs, so I don't think we need to mention it yet.

Reply:
What is the right way to track updating the docs with this info once .NET 5 ships?

Reply:
I created an issue [here](https://github.com/dotnet/docs/issues/16088) and labeled it with a new label, Release: .NET 5. @mairaw Please advise if I should do something different instead.